test/staking-rewards/StakingRewards.js

```diff
@@ -661,7 +664,7 @@ describe('StakingRewards', () => {
 
             const reward = await staking.pendingRewards(providerAddress);
 
-            const effectiveTime = BigNumber.min(now, programEndTime);
+            const effectiveTime = min(now, programEndTime);
```

Comment:
Minor: I'm pretty sure that you can use `Math.min` in all of those which take time values as input.